lib/shoryuken/manager.rb

```diff
@@ -12,7 +13,7 @@ class Manager
 
     def initialize(condvar)
       @count  = Shoryuken.options[:concurrency] || 25
-      @queues = Shoryuken.queues.dup.uniq
+      @polling_strategy = Shoryuken.options[:polling_strategy].new(Shoryuken.queues)
```

Comment:
We call this a polling strategy, but the class name that's actually used for the strategy isn't called a strategy at all. Nor is the namespace. This is confusing.